SwiftfulSourceControl/SettingsView.swift

```diff
@@ -8,6 +8,9 @@
 import SwiftUI
 
 struct SettingsView: View {
+    
+    @State private var doSomething: Bool = false
```

Comment:
Hey this works but we don't use this naming conventsion. Can you remame this to "isPreimum"